Source/WebCore/css/StyleProperties.cpp

```diff
@@ -575,7 +576,87 @@ static std::optional<CSSValueID> fontStretchKeyword(double value)
 
 String StyleProperties::fontValue() const
 {
-    // FIXME: This should check more font subproperties aand return empty string when they do not have their initial values. https://bugs.webkit.org/show_bug.cgi?id=247498
+    auto propertyReferenceKeywordValue = [&](const PropertyReference& property) {
+        auto* value = dynamicDowncast<CSSPrimitiveValue>(property.value());
+        return value ? value->valueID() : CSSValueInvalid;
+    };
+
+    auto propertyKeywordValue = [&](CSSPropertyID property) -> std::optional<CSSValueID> {
+        if (int foundPropertyIndex = findPropertyIndex(property); foundPropertyIndex != -1)
+            return propertyReferenceKeywordValue(propertyAt(foundPropertyIndex));
+        return std::nullopt;
+    };
```

Comment:
No, these are unique. There is a function for comparing with a specific value ID, but that’s not the same thing. I’d be open to more refactoring and reorganizing to be cleaner, but what you are seeing here is after multiple rounds of tightening and looking around for helper functions.

I am noting that `propertyReferenceKeywordValue` in particular is not something that needs to capture anything. Could just be a helper function rather than a lambda.

But the `propertyKeywordValue` function is something that maybe _should_ have been elsewhere in this file. Instead other places use wordy code with lots of local variables for this sort of thing.